config/conf/routes

```diff
@@ -18,11 +18,16 @@ PUT           /v1/devicegroups/:id                  @com.microsoft.azure.iotsolu
 
 DELETE        /v1/devicegroups/:id                  @com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.controllers.DeviceGroupController.deleteAsync(id)
 
-GET           /v1/packages                          @com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.controllers.PackageController.getAllAsync
-GET           /v1/packages/                         @com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.controllers.PackageController.getAllAsync
+GET           /v1/packages/configTypes              @com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.controllers.PackageController.getListAsync
+GET           /v1/packages/configTypes/             @com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.controllers.PackageController.getListAsync
+
+GET           /v1/packages/:type/:config     @com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.controllers.PackageController.getFilteredAsync(type,config)
```

Comment:
This needs to be accepting query params not values as part of url

Reply:
This has changed